app/services/querier.js

```diff
@@ -25,6 +25,19 @@ export default Service.extend(Filterizer, {
 
   isRunningQuery: alias('stompWebsocket.isConnected').readOnly(),
 
+  defaultAPIAggregation: computed('settings.defaultAggregation.{type,size}', function() {
+    return {
+      type: this.get('settings.defaultAggregation.type'),
+      size: this.get('settings.defaultAggregation.size')
+    };
```

Comment:
just return this.get('settings.defaultAggregation') ?

Reply:
👍